spec/acceptance/basic_posix_acl_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'posix_acl' do
+  context 'with default behaviour' do
+    it_behaves_like 'an idempotent resource' do
+      pp = <<-EOS
+      user { 'blub':}
+      file { '/opt/test':
+        ensure => directory,
+        owner  => root,
+        group  => root,
+        mode   => '2770',
+      }
+      -> posix_acl { '/opt/test':
```

Comment:
Reading this I wonder if it should autorequire the file with the same path.

Reply:
I had the same idea. I've that on my todo list (maybe that's even already implemented. I didn't check it yet, code comes from a customer)

Reply:
I will work on it on https://github.com/voxpupuli/puppet-posix_acl/pull/106